build.sh

```diff
@@ -5,6 +5,8 @@ then
   git clone --branch 3.0.1 https://github.com/AltaPay/sdk-php.git
   cp -rf sdk-php/lib/* lib/altapay/altapay-php-sdk/lib/
   rm -rf sdk-php/
+  composer install
+  composer dump-autoload -a
```

Comment:
optimize and avoid test tools
```suggestion
  composer install -a -o --no-dev
```